### ISSUE TITLE
Getting `error: request failed or timed out`

### ISSUE THREAD

**Issue by shubham-spreadd** [Web3Auth/web3auth-backend#10]
I've checked for the implementation field with the web3auth team. 
Similar implementation I've added for a prod but getting an initialization error for the same while executing connect method for node js!
 
 
 My Configuration: 
```
 const web3auth = new Web3Auth({
  web3AuthNetwork: 'aqua',
  clientId: web3Auth.clientId,
  chainConfig: {
    chainNamespace: CHAIN_NAMESPACES.EIP155,
    chainId: `web3Auth.chainId`,
    rpcTarget: `https://bsc-dataseed.binance.org`,
  },
});

web3auth.init({});
```

**Comment by chaitanyapotti** [comment]
Pls share console logs..

**Comment by chaitanyapotti** [comment]
This should be fixed in 3.1.0
Pls upgrade
